go/vt/mysqlctl/schema.go

```diff
@@ -152,6 +104,10 @@ func (mysqld *Mysqld) GetSchema(dbName string, tables, excludeTables []string, i
 		sd.TableDefinitions = append(sd.TableDefinitions, td)
 	}
 
+	sd, err = sd.FilterTables(tables, excludeTables, includeViews)
+	if err != nil {
+		return nil, err
+	}
 	sd.GenerateSchemaVersion()
```

Comment:
Isn't FilterTables already doing GenerateSchemaVersion()? Any reason to do it twice?